README.md

```diff
@@ -148,9 +137,6 @@ for details on how to configure ECR policies
 
 Use the action to output your Docker credentials for logging into ECR Private, then use the credentials to run your private image as a service in another job.
 
-> [!WARNING]
```

Comment:
Can you change the example for `Run an image as a service` to set `mask-password` to false, and keep this warning? This example won't work without it being set to false.

Reply:
Added `mask-password` to false and updated warning.